docker-compose.yml

```diff
@@ -16,7 +16,6 @@ services:
     command: sbin/uta-extract /ncbi-dir /uta-extract/work /uta-extract/logs
     volumes:
       - ${UTA_ETL_NCBI_DIR}:/ncbi-dir
-      - ${UTA_ETL_SEQREPO_DIR}:/usr/local/share/seqrepo
```

Comment:
SeqRepo is not used by any of the "extract" scripts. I think this is left over from when I moved `exonset-to-seqanno` to the extract step.